src/Avalonia.Controls/IScrollAnchorProvider.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Avalonia.Controls
+{
+    public interface IScrollAnchorProvider
```

Comment:
This interface doesn't seem to be used anywhere, do we need it merged without implementation?

Reply:
I think we'll need to implement this soon, I left it in there but unimplemented so that I could at least port most of the code on the `ItemsRepeater` side that relates to scroll anchors.